src/actions/BulkSetRackNoAction.php

```
@@ -0,0 +1,24 @@
+namespace hipanel\modules\server\actions;
+
+use yii\helpers\ArrayHelper;
+
+class BulkSetRackNoAction extends AssignableHubs
+{
+    protected function collectFromRequest(): array
+    {
+        $hubs = $this->controller->request->post($this->collection->getModel()->formName());
+        $rackId = ArrayHelper::remove($hubs, 'rack_id');
+        $rackPort = ArrayHelper::remove($hubs, 'rack_port');
+        foreach ($hubs as $id => $model) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$model'. (Unused Code Rules)

(UnusedLocalVariable)